### PR TITLE
Add an appdata file

### PR DESCRIPTION
**Pull request by sanjayankur31**
This was reported as https://sourceforge.net/p/pvqt/feature-requests/5/

### CODE REVIEW

File: panini.appdata.xml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      Panini can do those things because it paints the picture on a three dimensional
+      surface, either a sphere or a cylinder, which you then view in perspective.
+      Shifting the point of view changes the apparent perspective of the image, and
+      other controls let you frame the view to your liking.  Then you can save the
+      screen image to a file at higher-than-screen resolution.
+    </p>
+  </description>
+  <!-- no screenshots -->
+  <url type="homepage">https://lazarus-pkgs.github.io/lazarus-pkgs/panini.html</url>
+  <updatecontact>tksharpless@gmail.com</updatecontact>
```
Comment (jubalh) [inline panini.appdata.xml]: Thomas isn't the main contact for this anymore, please add my email instead (you can see it in `git log`).

**Comment by jubalh** [comment]
Nice, thanks!
Shouldnt there be a change in the project file to install this appdata file too?

File: panini.appdata.xml (hunk 1 above)
Comment (sanjayankur31) [reply panini.appdata.xml]: Done, thanks.

**Comment by sanjayankur31** [comment]
Yeh, ideally, the project file should put it in `%{_datadir}/metainfo/`, where `%{_datadir}` usually expands to `/usr/share`. I can look into updating the pro file too. Should I? :)

This is the Fedora documentation on Appdata, for reference: https://fedoraproject.org/wiki/Packaging:AppData

File: linux/panini.appdata.xml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      Panini can do those things because it paints the picture on a three dimensional
+      surface, either a sphere or a cylinder, which you then view in perspective.
+      Shifting the point of view changes the apparent perspective of the image, and
+      other controls let you frame the view to your liking.  Then you can save the
+      screen image to a file at higher-than-screen resolution.
+    </p>
+  </description>
+  <!-- no screenshots -->
+  <url type="homepage">https://lazarus-pkgs.github.io/lazarus-pkgs/panini.html</url>
+  <updatecontact>jubalh AT iodoru DOT org</updatecontact>
```
Comment (jubalh) [inline linux/panini.appdata.xml]: is it okay to write the email in such a format?

**Comment by jubalh** [comment]
Please squash the first 3 commits into one.

File: linux/panini.appdata.xml (hunk 2 above)
Comment (sanjayankur31) [reply linux/panini.appdata.xml]: I don't think the e-mail address is parsed at all currently. I did validated it and didn't get an error:

```
appstream-util validate-relax linux/panini.appdata.xml 
linux/panini.appdata.xml: OK
```

So I reckon it's OK.

Comment (jubalh) [reply linux/panini.appdata.xml]: Ok

**Comment by jubalh** [comment]
Thanks for this!

**Pull request merged**